software/Test_Robot_Software/segment_module/Segment.egg-info/PKG-INFO

```diff
@@ -0,0 +1,3 @@
+Metadata-Version: 2.1
```

Comment:
The whole Segment.egg-info and build folders should not be uploaded

Reply:
Removed the segment.egg-info and build folders